www-client/elinks/elinks-0.13_p20180225.ebuild

```diff
@@ -0,0 +1,137 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python2_7 )
+PYTHON_REQ_USE="threads"
+inherit autotools python-any-r1 vcs-snapshot
```

Comment:
Since you know the commit id, please don't use `vcs-snapshot` and override `S` correctly instead.